04-17 10:47:57 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1333)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1308)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:164)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnection.createStatement(PooledConnection.java:334)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.testConnection(DataSourcePool.java:496)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.checkDataSource(DataSourcePool.java:356)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager.checkDataSource(DataSourceManager.java:228)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager.access$100(DataSourceManager.java:37)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager$Checker.run(DataSourceManager.java:240)
	at com.avaje.ebeaninternal.server.lib.BackgroundThread$Runner.runJobs(BackgroundThread.java:208)
	at com.avaje.ebeaninternal.server.lib.BackgroundThread$Runner.run(BackgroundThread.java:172)
	at java.lang.Thread.run(Unknown Source)
04-17 10:47:57 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1333)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1308)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:164)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnection.createStatement(PooledConnection.java:334)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.testConnection(DataSourcePool.java:496)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.validateConnection(DataSourcePool.java:529)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnection.close(PooledConnection.java:505)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.checkDataSource(DataSourcePool.java:370)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager.checkDataSource(DataSourceManager.java:228)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager.access$100(DataSourceManager.java:37)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager$Checker.run(DataSourceManager.java:240)
	at com.avaje.ebeaninternal.server.lib.BackgroundThread$Runner.runJobs(BackgroundThread.java:208)
	at com.avaje.ebeaninternal.server.lib.BackgroundThread$Runner.run(BackgroundThread.java:172)
	at java.lang.Thread.run(Unknown Source)
04-17 10:47:58 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1333)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1308)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:164)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnection.createStatement(PooledConnection.java:334)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.testConnection(DataSourcePool.java:496)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.validateConnection(DataSourcePool.java:529)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnection.close(PooledConnection.java:505)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnectionQueue.closeBusyConnection(PooledConnectionQueue.java:515)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnectionQueue.closeBusyConnections(PooledConnectionQueue.java:491)
	at com.avaje.ebeaninternal.server.lib.sql.PooledConnectionQueue.shutdown(PooledConnectionQueue.java:360)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourcePool.shutdown(DataSourcePool.java:694)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceManager.shutdown(DataSourceManager.java:167)
	at com.avaje.ebeaninternal.server.lib.sql.DataSourceGlobalManager.shutdown(DataSourceGlobalManager.java:45)
	at com.avaje.ebeaninternal.server.lib.ShutdownManager.shutdown(ShutdownManager.java:183)
	at com.avaje.ebeaninternal.server.lib.ShutdownHook.run(ShutdownHook.java:47)
04-17 14:13:31 database: close
java.lang.NullPointerException
	at org.h2.store.PageLog.removeUntil(PageLog.java:720)
	at org.h2.store.PageLog.removeUntil(PageLog.java:697)
	at org.h2.store.PageStore.checkpoint(PageStore.java:408)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1121)
	at org.h2.engine.Database.close(Database.java:1074)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:80)
